App/Redux/PhotoViewingSelectors.ts

```diff
@@ -1,13 +1,14 @@
 import { RootState } from './Types'
 import { ThreadData } from './PhotoViewingRedux'
+import { Thread } from '../Models/TextileTypes'
 
 export function defaultThreadData (state: RootState): ThreadData | undefined {
   return Object.keys(state.photoViewing.threads)
     .map((key) => state.photoViewing.threads[key] )
     .find((threadData) => threadData.thread.name === 'default')
 }
 
-export function threadDataByThreadId (state: RootState, id: string) {
+export function threadDataByThreadId (state: RootState, id: string): ThreadData {
```

Comment:
You can leave these return types I guess, but they are not needed since they are inferred by the TS compiler.

Reply:
They help me reason about what's being returned, so I left them in... make it more readable to me?

Reply:
Totally, it's fine to leave.